certbot-apache/setup.py

```diff
@@ -11,7 +11,7 @@
     'acme=={0}'.format(version),
     'certbot=={0}'.format(version),
     'mock',
-    'python-augeas',
+    'python-augeas==0.5.0',
```

Comment:
nit: We should probably do `python-augeas<=0.5.0` for the older distros that are using an even older version of the package.